src/ansible_builder/constants.py

```diff
@@ -13,15 +13,15 @@
 base_roles_path = '/usr/share/ansible/roles'
 base_collections_path = '/usr/share/ansible/collections'
 
-build_arg_defaults = dict(
+build_arg_defaults = {
```

Comment:
Unrelated change, but ok with it. I think there may be a pylint check code for this, actually? Might want to enable it.